articles/server-platforms/golang/01-login.md

```diff
@@ -11,13 +11,16 @@ This tutorial and seed project have been tested with the following:
 :::
 
 <%= include('../../_includes/_package', {
-  pkgRepo: 'auth0-golang',
+  pkgOrg: 'auth0-samples',
+  pkgRepo: 'auth0-golang-samples',
+  githubUrl:'https://github.com/auth0-samples/auth0-golang-samples/tree/master/00-Starter-Seed',
```

Comment:
@mehreencs87 Looks good, one last change please. Since this is the regular web app, can you please set the `githubUrl` value to `https://github.com/auth0-samples/auth0-golang-samples/tree/master/00-Starter-Seed/regular-web-app`? Thanks.